client/components/data/query-site-domains/README.md

```diff
@@ -0,0 +1,32 @@
+Query SiteDomains
+===========================
+
+`<QuerySiteDomains />` is a React component used in managing network requests for sites/%s/domains.
```

Comment:
I'm not sure because we have some Domains stuff that don't dependes of a site and I wonder if we could have `state.domains` and in that case maybe we could have a `<QueryDomains>` which won't depend of a site.